validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java

```diff
@@ -270,7 +276,15 @@ public void loadCredentials() {
       if (!hasAuthType()) {
         throw new RuntimeException("Credential cloud.auth_type definition missing");
       }
-      for (String keyFile : ALL_KEY_FILES) {
+      String authType = getAuthType();
+      Set<String> keyFiles;
+      if (authType.equals(ES_CERT_TYPE) || authType.equals(RSA_CERT_TYPE))
+      {
```

Comment:
opening brace should be on previous line

Reply:
Done.